### PR TITLE
welle-io: init at 1.0-rc1

### PR DESCRIPTION
**Pull request by ck3d**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -14909,6 +14909,8 @@ with pkgs;
 
   wavrsocvt = callPackage ../applications/misc/audio/wavrsocvt { };
 
+  welle-io = qt5.callPackage ../applications/misc/welle-io { };
```
Comment (orivej) [inline pkgs/top-level/all-packages.nix]: Please use `libsForQt5.callPackage`. It is different from `qt5.callPackage` , which it is not meant for use outside of the definition of Qt.

File: pkgs/applications/misc/welle-io/default.nix [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    qtbase
+    qtcharts
+    qtmultimedia
+    qtquickcontrols
+    qtquickcontrols2
+    rtl-sdr
+  ];
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+
```
Comment (orivej) [inline pkgs/applications/misc/welle-io/default.nix]: I'd appreciate if attributes were in the familiar order: nativeBuildInputs, buildInputs, cmakeFlags, enableParallelBuilding (which is now the default for cmake, qmake and meson projects).

**Comment by ck3d** [comment]
Thanks a lot for reviewing. I updated the PR.

**Comment by orivej** [comment]
Thank you!

**Pull request merged**